electrum/wallet.py

```diff
@@ -1333,12 +1336,13 @@ def make_unsigned_transaction(
         outputs = copy.deepcopy(outputs)
 
         # check outputs
-        i_max = None
+        i_max = []
+        i_max_sum = 0
         for i, o in enumerate(outputs):
-            if o.value == '!':
-                if i_max is not None:
-                    raise MultipleSpendMaxTxOutputs()
-                i_max = i
+            if check_max_spend(o.value):
+                x = parse_max_spend(o.value)
+                i_max_sum += x
+                i_max.append((x,i))
```

Comment:
If using the changed `parse_max_spend` I suggest in another comment,
you could instead do
```suggestion
            weight = parse_max_spend(o.value)
            if weight:
                i_max_sum += weight
                i_max.append((weight, i))
```